### DISCUSSION TITLE
Inception

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#324] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/inception/

**Comment by brunoaugusto** [comment]
Reconstrução? :thinking:

**Comment by vish-blog** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/inception/#comment_1128952392>):

> Reconstrução? 🤔

A tirinha é bem antiga e a original era em coreano e estava com uma qualidade mais do que péssima. A melhor que eu achei — de um site indonésio — ainda estava muito ruim então eu refiz, quadrinho por quadrinho, e repintei o melhor que pude.
